apps/dashboard/app/views/batch_connect/sessions/connections/_resolution_field.html.erb

```diff
@@ -0,0 +1,77 @@
+<div id="<%= id.to_s + "_group"%>" class="form-group">
+  <%= form.label(id, opts[:label]) %>
+  <%= form.hidden_field(id, id: "#{id}_field") %>
+  <div class="row">
+    <div class="col-sm-6">
+      <div class="input-group">
+        <div class="input-group-addon" style="width: 70px">width</div>
+        <input class="form-control" id="<%= id.to_s + "_x_field" %>" min="100" required="<%= opts[:required] %>"/>
+        <div class="input-group-addon">px</div>
+      </div>
+    </div>
+    <div class="col-sm-6">
+      <div class="input-group">
+        <div class="input-group-addon" style="width: 70px">height</div>
+        <input class="form-control" id="<%= id.to_s + "_y_field" %>"
+        min="100" required="<%= opts[:required] %>"/>
+        <div class="input-group-addon">px</div>
+      </div>
+    </div>
+  </div>
+  <% if opts[:help] %>
+    <span class="help-block"><%= opts[:help] %></span>
+  <% end %>
+  <button name="button" type="button" id="<%= id.to_s + "_reset" %>" class="btn btn-default"><%= t('dashboard.batch_connect_form_reset_resolution') %></button>
+  <script type="text/javascript">
+    <<-EOT.html_safe

```

Comment:
Remove this, this line is Ruby code and not JavaScript.

Reply:
Just to make sure are you talking about line 26?